### PR TITLE
[Fix-802] Widget-download-element

### PR DESCRIPTION
**Pull request by MashaPav**

### CODE REVIEW

**Comment by MashaPav** [comment]
@yuvalbl tested on all widgets on staging preview, looks good to me. 

@daniel-ilan can you double-check me pls?

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "env-cmd": "^10.1.0",
     "file-saver": "^2.0.5",
     "html-to-image": "^1.6.2",
```
Comment (yuvalbl) [inline package.json:17]: do we still need `html-to-image`?

Comment (MashaPav) [reply package.json]: @yuvalbl no, it's not needed. I removed it

**Comment by MashaPav** [comment]
opened a new one https://github.com/hasadna/anyway-newsflash-infographics/pull/847

**Pull request closed without merging**